File: Backend/lib/src/main/kotlin/com/github/bratek20/ostium/gamesetup/context/Impl.kt
package com.github.bratek20.ostium.gamesetup.context

import com.github.bratek20.architecture.context.api.ContextBuilder
import com.github.bratek20.architecture.context.api.ContextModule

import com.github.bratek20.ostium.gamesetup.api.*
import com.github.bratek20.ostium.gamesetup.impl.*

class GameSetupImpl: ContextModule {
    override fun apply(builder: ContextBuilder) {
        builder
            .setImpl(GameSetupApi::class.java, GameSetupApiLogic::class.java)
    }
}

File: Frontend/Logic/B20/Frontend/Timer/Context/Impl.cs
using B20.Architecture.Contexts.Api;
using B20.Frontend.Timer.Api;
using B20.Frontend.Timer.Impl;

namespace B20.Frontend.Timer.Context
{
    public class TimerImpl: ContextModule
    {
        public void Apply(ContextBuilder builder)
        {
            builder
                .SetImpl<TimerApi, TimerApiLogic>();
        }
    }
}